app/gui/view/component-browser/component-group/src/lib.rs

```diff
@@ -65,6 +75,19 @@ pub mod background {
     }
 }
 
+/// The transparent overlay over Component Group View Header, used for capturing mouse events.
+pub mod header_overlay {
+    use super::*;
+
+    ensogl_core::define_shape_system! {
+        above = [background];
+        () {
+            let bg_color = Rgba::new(0.0,0.0,0.0,0.000_001);
```

Comment:
There was a constant for that – please use it instead, as this looks like a magic :D

Reply:
I cannot find it. There is only `transparent` method of `Rgba`, but there the alpha is 0.0. Shall I create such constant (or `Rgba`'s static method)?

Reply:
<img width="625" alt="image" src="https://user-images.githubusercontent.com/1623053/165511728-8cd4a67d-7f41-442f-8467-36512729e463.png">

Reply:
Pfff, why _red_? I was looking for black :P